app.py

```diff
@@ -147,13 +162,63 @@ def graph_refresh() -> None:
             return
 
 
+def graph_refresh_package_analyzer() -> None:
+    """Schedule refresh for packages that are not yet analyzed by package analyzer."""
+    packages = _GRAPH_DB.retrieve_unanalyzed_python_package_versions()
+
+    if not packages:
+        _LOGGER.info("No unanalyzed packages found")
+        return
+
+    count = 0
+    for item in packages:
+        package = item["package_name"]
+        version = item["package_version"]
+        url = item["index_url"]
+        try:
+            analysis_id = openshift.schedule_package_analyzer(
```

Comment:
Variable `openshift` is not declared in this scope. You can promote openshift from `graph_refresh_solver` function to a global variable `_OPENSHIFT` as you did with `GRAPH_DB`.